cmsplugin_filer_image/migrations/0013_mv_thumbnailoption_to_filer.py

```diff
@@ -0,0 +1,240 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+from cmsplugin_filer_utils.migration import rename_tables_new_to_old
+
+class Migration(DataMigration):
+    
+    dependencies = [("filer", "0003_thumbnailoption")]
+    
+    cms_plugin_table_mapping = (
+        ('cmsplugin_filerfolder', 'cmsplugin_filer_folder_filerfolder'),
+    )
+
+    def forwards(self, orm):
+        from filer.models import ThumbnailOption
```

Comment:
@stefanfoulis it's a kind of a hack we used because ``orm`` gets confused. ``orm.ThumbnailOption`` (for cmspluginfiler) and ``orm['filer.ThumbnailOption']`` (for filer) apparently conflicts (probably  due to the model having the same name)